spec/offchain.tex

```diff
@@ -185,45 +189,75 @@ \subsubsection{Processing transactions off-chain}
 receiving party $\party_{i}$ is the next snapshot
 leader, a message to request snapshot signatures $\hpRS$ is sent. \\
 
-\dparagraph{$\hpRD$.}\quad Upon receiving request $(\hpRD,\tx)$, the transaction is
-checked against the \emph{local} ledger state. If decommit is not applicable yet
-or another decommit is still pending, the protocol does $\KwWait$ to retry later or
-eventually marks the decommit as invalid. In case there is no snapshot in flight
-and party is the leader then a snapshot request $\hpRS$ is
-sent containing the decommit transaction $\tx_{\omega}$. \\
+\dparagraph{$\hpRD$.}\quad Upon receiving request $(\hpRD,\tx_\omega)$, \red{if
+  another decrement (or increment) is pending to be settled the transaction is
+  rejected. Otherwise} the transaction is checked against the \emph{local}
+  ledger state and if it is not applicable yet or another decommit is still, the protocol does $\KwWait$ to
+  retry later or eventually marks the decommit as invalid. \red{After applying $\tx$, it's inputs
```

Comment:
Should: fix this sentence. It was already mentioning pending decommits. Let's just extend the sentence there.

```suggestion
\dparagraph{$\hpRD$.}\quad Upon receiving request $(\hpRD,\tx_\omega)$, the transaction is checked against the \emph{local}
  ledger state and if it is not applicable yet or another commit or decommit is pending still, the protocol does $\KwWait$ to
  retry later or eventually marks the decommit as invalid. \red{After applying $\tx$, it's inputs
```